components/clp-py-utils/clp_py_utils/initialize-clp-metadata-db.py

```diff
@@ -55,6 +57,10 @@ def main(argv):
         ) as metadata_db_cursor:
             if StorageEngine.CLP_S == storage_engine:
                 create_datasets_table(metadata_db_cursor, table_prefix)
+                # Note: the webui still expect tables with default dataset
```

Comment:
There is still a small issue: the CLP_DEFAULT_DATASET_NAME is not added to the dataset table yet.
But I feel that can be delayed until an archive is actually compressed. If you prefer, we can also explicility add the CLP_DEFAULT_DATASET_NAME to the datasets table